admob/integration_test/src/integration_test.cc

```diff
@@ -489,6 +506,229 @@ TEST_F(FirebaseAdMobTest, TestRequestConfigurationSetGet) {
                          retrieved_configuration.test_device_ids.end(), "3"));
 }
 
+// Simple Load Tests as a sanity check. These don't show the ad, just
```

Comment:
Moved some BannerView and Interstitial tests around.  Will call out the new tests via comments below. so that you needed waste your time reviewing unchanged tests.